conjur/api/api.py

```diff
@@ -109,12 +109,13 @@ def login(self, login_id=None, password=None):
 
         return self.api_key
 
-    def authenticate(self):
+    def authenticate(self) -> requests.Response:
```

Comment:
return str 
look it return invoke_endpoint(HttpVerb.POST, ConjurEndpoint.AUTHENTICATE, params,
                               self.api_key, ssl_verify=self._ssl_verify)**.text**

Reply:
Done